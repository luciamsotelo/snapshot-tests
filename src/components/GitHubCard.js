// GitHubCard.js
import React from "react";
import { Card } from "react-bootstrap";

function GitHubCard() {
    return (
        <Card style={{ width: "18rem" }}>
            {/* Add your GitHub username */}
            <Card.Title>GitHub Lucia</Card.Title>

            {/* Add a photo of you */}
            <Card.Img variant="top" src="https://plus.unsplash.com/premium_photo-1661943864527-d714736dfd16?" alt="Your Photo" />

            {/* Add a blurb about you */}
            <Card.Body>
                <Card.Text>
                    This is a blurb about me. I am a passionate developer who
                    loves coding.
                </Card.Text>
            </Card.Body>
        </Card>
    );
}

export default GitHubCard;
