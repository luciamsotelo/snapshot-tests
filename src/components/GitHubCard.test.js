// GitHubCard.test.js
import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import GitHubCard from './GitHubCard';

test('GitHubCard snapshot', () => {
  const { asFragment } = render(<GitHubCard />);
  expect(asFragment()).toMatchSnapshot();
});

// Optional: Uncomment the following lines if you want to use react-test-renderer directly
// test('GitHubCard snapshot using react-test-renderer', () => {
//   const tree = renderer.create(<GitHubCard />).toJSON();
//   expect(tree).toMatchSnapshot();
// });
