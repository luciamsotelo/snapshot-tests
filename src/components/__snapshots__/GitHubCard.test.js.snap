// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHubCard snapshot 1`] = `
<DocumentFragment>
  <div
    class="card"
    style="width: 18rem;"
  >
    <div
      class="card-title h5"
    >
      GitHub Lucia
    </div>
    <img
      alt="Your Photo"
      class="card-img-top"
      src="https://plus.unsplash.com/premium_photo-1661943864527-d714736dfd16?"
    />
    <div
      class="card-body"
    >
      <p
        class="card-text"
      >
        This is a blurb about me. I am a passionate developer who loves coding.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
